### PR TITLE
Get case studies running on the wildcard semantics

### PR DESCRIPTION
**Pull request by Hjdskes**
I will write instructions later tonight.

### CODE REVIEW

File: stratego/case-studies/Main.hs [hunk 1]
```diff
@@ -23,137 +22,27 @@ import           Data.Foldable
...
--- wittnesses (W.NumberLiteral n) (W.NumberLiteral n') = n == n'
--- wittnesses _ _ = False
-
--- for :: Functor f => f a -> (a -> b) -> f b
--- for = flip fmap
-
--- orElse :: Maybe a -> a -> a
--- orElse = flip fromMaybe
-
 measure :: String -> IO () -> IO ()
```
Comment (Hjdskes) [inline stratego/case-studies/Main.hs]: Do you still want to measure this? Otherwise I should remove this function.

Comment (Hjdskes) [reply stratego/case-studies/Main.hs]: The same goes for the line `_ <- CM.measure (CT.nfIO (return terms)) 1` in `caseStudy` below.

**Comment by Hjdskes** [comment]
Below follows the description of the relevant files and the instructions to run the case studies. I decribed how I run the case studies (i.e., using Stack), I don't know about Cabal or Nix. Should I also include how to change the fuel for the case studies? Right now this is just a random `5`.

---

The case studies are provided in the folder `stratego/case-studies`. The Haskell source file `Main.hs` contains the code that runs each case study with the wildcard semantics.

Each case study is found in its own subfolder in `stratego/case-studies`, where the file with the `*.str` suffix contains the actual Stratego code -- all other files are generated from this file (using `stratego/case-studies/prepare.sh`) and can be ignored for this purpose.

To run the case studies, first change directory to the toplevel sturdy source directory. Then build the `sturdy-stratego` project using `stack build sturdy-stratego`, and run the case studies using `stack exec case-studies`.

File: stratego/case-studies/Main.hs (hunk 1 above)
Comment (svenkeidel) [reply stratego/case-studies/Main.hs]: It sets up a timer of how long the case studies run.

**Comment by svenkeidel** [comment]
No that is fine thank you. I'll add these instructions in a readme.

File: stratego/case-studies/Main.hs (hunk 1 above)
Comment (Hjdskes) [reply stratego/case-studies/Main.hs]: It's unused at the moment.

**Pull request merged**